cfg.yaml

```diff
@@ -9,14 +9,18 @@ db-verify-interval: 1   #  How often to check the DB size. By default Postee che
 
 # Routes are used to define how to handle an incoming message
 routes:
-- name: route1                                 #  Route name. Must be unique
```

Comment:
I would keep it as example

Reply:
this `route` was commented as an example.